pkg/codegen/internal/test/testdata/simple-methods-schema/dotnet/Utilities.cs

```diff
@@ -0,0 +1,87 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.IO;
+using System.Reflection;
+using Pulumi;
+
+namespace Pulumi.Example
+{
+    static class Utilities
+    {
+        public static string? GetEnv(params string[] names)
+        {
+            foreach (var n in names)
+            {
+                var value = Environment.GetEnvironmentVariable(n);
+                if (value != null)
+                {
+                    return value;
+                }
+            }
+            return null;
+        }
+
+        static string[] trueValues = { "1", "t", "T", "true", "TRUE", "True" };
+        static string[] falseValues = { "0", "f", "F", "false", "FALSE", "False" };
+        public static bool? GetEnvBoolean(params string[] names)
```

Comment:
Interesting what this file is doing here :) Some artifact of codegen I assume.

Reply:
We just always emit this file in full for SDKs, and don't filter out its members if they aren't used.